bucket/rom-properties-np.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "1.9",
+    "description": "This shell extension adds a few nice features to file browsers for managing video game ROM and disc images.",
+    "homepage": "https://github.com/GerbilSoft/rom-properties",
+    "license": "GPL-2.0",
+    "url": "https://github.com/GerbilSoft/rom-properties/releases/download/v1.9/rom-properties_1.9-windows.zip",
+    "hash": "f8a788700034b52bf007f58eec595e49892452366e72cf8ded68d8f9cb9dabdd",
+    "pre_install": [
+        "if (!(is_admin)) {",
+        "    error 'Administrator rights are required to install $app.'",
+        "    exit 1",
+        "}"
+    ],
```

```suggestion
    "pre_install": "if (!(is_admin)) { error \"Administrator rights are required to install $app\"; break }",
```

If the user is installing/upgrading multiple apps, `exit` would cause the whole process to stop, while `break` only skips installation of this app.

Variables will not work inside **single quotes**. The error message would become `ERROR Administrator rights are required to install $app.`, which is not what we want.